### PR TITLE
PR number 2

### PR DESCRIPTION
**Pull request by spajic**
Handle code review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +135,59 @@ Or install it yourself as:
...
+- Capture maximum 1 interaction in `:controller` spec (see https://github.com/rails/rails/issues/13851)
+- Only support `application/json` interactions yet
 
 ## Development
+To run cucumber specs:
+
+```bash
+bundle exec appraisal install
+bundle exec appraisal rails-5 cucumber
+rm -r spec/murker && bundle exec appraisal reils-5 cucumber
```
Comment (aderyabin) [inline README.md]: typo

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
 language: ruby
 rvm:
-  - 2.4.3
-before_install: gem install bundler -v 1.16.0
+  - 2.3.2
+before_install:
+  - gem install bundler -v 1.16.1
+script:
+  - RAILS_ENV=test bundle exec rake spec
+  - bundle exec appraisal install && RAILS_ENV=test bundle exec appraisal rails-5 cucumber
```
Comment (aderyabin) [inline .travis.yml]: Thoughtbot recommends to use multiple gemfiles: https://github.com/thoughtbot/appraisal#travis-ci-integration

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+AllCops:
```
Comment (aderyabin) [inline .rubocop.yml:1]: – Need to add rubocop usage to CI
– I have a lot of alerts from rubocop locally

File: Appraisals [hunk 4]
```diff
@@ -0,0 +1,7 @@
+appraise 'rails-5' do
+  source 'https://rubygems.org'
+
+  gem 'rails', '5.1.4'
+  gem 'sqlite3'
```
Comment (aderyabin) [inline Appraisals]: Do we really need database usage (ActiveRecord?) to test requests? I think you can use stubs.

File: README.md (hunk 1 above)
Comment (spajic) [reply README.md]: fixed

File: .travis.yml (hunk 2 above)
Comment (spajic) [reply .travis.yml]: fixed

File: .rubocop.yml (hunk 3 above)
Comment (spajic) [reply .rubocop.yml:1]: I have setup rubocop on TravisCI - got no warnings

File: Appraisals (hunk 4 above)
Comment (spajic) [reply Appraisals]: Eliminated `sqlite` and `ActiveRecord`

**Comment by aderyabin** [comment]
Could you please squash all commits into one?

**Comment by spajic** [comment]
@aderyabin Yes, opened #3 with one squashed commit

**Comment by spajic** [comment]
Close in favour of #3

**Pull request closed without merging**